bundles/mapping/mapuserlayers/domain/UserLayerModelBuilder.js

```diff
@@ -22,6 +22,20 @@ Oskari.clazz.define(
             // call parent parseLayerData
             this.wfsBuilder.parseLayerData(layer, mapLayerJson, maplayerService);
             layer.setLocale(mapLayerJson.locale);
+
+            const values = {
+                locale: layer.getLocale(),
+                style: layer.getCurrentStyle().getFeatureStyle(),
+                id: layer.getId()
+            };
```

Comment:
We should only send layer id in the request since at least the locale can change after the model builder has been run and this would create an old layer name be shown when activating this tool.